tests/components/media_player/test_init.py

```diff
@@ -80,11 +81,21 @@ async def test_get_image_http_remote(hass, aiohttp_client):
         client = await aiohttp_client(hass.http.app)
 
         with patch(
-            "homeassistant.components.media_player.MediaPlayerDevice."
+            "homeassistant.components.media_player.MediaPlayerEntity."
             "async_get_media_image",
             return_value=(b"image", "image/jpeg"),
         ):
             resp = await client.get(state.attributes["entity_picture_local"])
             content = await resp.read()
 
         assert content == b"image"
+
+
+def test_deprecated_base_class(caplog):
+    """Test deprecated base class."""
+
+    class CustomMediaPlayer(media_player.MediaPlayerDevice):
+        pass
+
+    CustomMediaPlayer()
+    assert "MediaPlayerDevice is deprecated, modify CustomMediaPlayer" in caplog.text
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5418277)